docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -28,6 +29,256 @@
+		* Removed property: `days_to_comorbidity`
+		* Removed property: `days_to_risk_factor`
+		* Removed property: `diabetes_treatment_type`
+		* Removed property: `dlco_ref_predictive_percent`
+		* Removed property: `eye_color`
+		* Removed property: `fev1_fvc_post_bronch_percent`
+		* Removed property: `fev1_fvc_pre_bronch_percent`
+		* Removed property: `fev1_ref_post_bronch_percent`
+		* Removed property: `fev1_ref_pre_bronch_percent`
+		* Removed property: `haart_treatment_indicator`
+		* Removed property: `height`
+		* Removed property: `hepatitis_sustained_virological_response`
+		* Removed property: `hiv_viral_load`
+		* Removed property: `hormonal_contraceptive_type`
+		* Removed property: `hormonal_contraceptive_use`
+		* Removed property: `hpv_positive_type`
+		* Removed property: `hysterectomy_margins_involved`
+		* Removed property: `hysterectomy_type`
+		* Removed property: `immunosuppressive_treatment_type`
+		* Removed property: `menopause_status`
+		* Removed property: `nadir_cd4_count`
+		* Removed property: `pancreatitis_onset_year`
+		* Removed property: `pregnancy_count`
+		* Removed property: `pregnancy_outcome`
+		* Removed property: `reflux_treatment_type`
+		* Removed property: `risk_factor`
+		* Removed property: `risk_factor_method_of_diagnosis`
+		* Removed property: `risk_factor_treatment`
+		* Removed property: `risk_factors`
+		* Removed property: `undescended_testis_corrected`
+		* Removed property: `undescended_testis_corrected_age`
+		* Removed property: `undescended_testis_corrected_laterality`
+		* Removed property: `undescended_testis_corrected_method`
+		* Removed property: `undescended_testis_history`
+		* Removed property: `undescended_testis_history_laterality`
+		* Removed property: `viral_hepatitis_serologies`
+		* Removed property: `weight`
+		* Changes made to `days_to_first_event`
+		* Changes made to `timepoint_category`
+			* New permissible value: `After Study Registration`
+* Altered `read_group` Entity
+	* Changes made to `deprecated`
+		* Removed permissible value: `RIN`
+	* Changes made to `properties`
+		* Changes made to `single_cell_library`
+			* New permissible value: `Chromium 3' Gene Expression v1 Library`
+			* New permissible value: `Chromium 3' Gene Expression v4 Library`
+			* New permissible value: `Chromium Next GEM Single Cell Multiome ATAC + Gene Expression Library`
+		* Changes made to `target_capture_kit`
+			* New permissible value: `xGen Exome Research Panel v2.0`
+* Altered `sample` Entity
+	* Changes made to `deprecated`
+		* New permissible value: `tumor_code_id`
+	* Changes made to `properties`
+		* Removed property: `composition`
+		* Removed property: `is_ffpe`
+		* Removed property: `oct_embedded`
+		* Removed property: `sample_type_id`
+		* Removed property: `tumor_code`
+		* Changes made to `biospecimen_anatomic_site`
+			* Removed permissible value: `Brain Stem`
+			* Removed permissible value: `Common Duct`
+			* Changes made to `deprecated_enum`
```

Comment:
no idea if we want to edit this a bit

Reply:
Let's delete this line and the two under it.  Not needed for end-users.